docs/py/timeseries_forecaster.py

```diff
@@ -0,0 +1,67 @@
+"""shell
+pip install autokeras
+"""
+
+"""
+To make this tutorial easy to follow, we use the UCI Airquality dataset, and try to
+forecast the AH value at the different timesteps. Some basic preprocessing has also
+been performed on the dataset as it required cleanup.
+
+## A Simple Example
+The first step is to prepare your data. Here we use the [UCI Airquality dataset]
+(https://archive.ics.uci.edu/ml/datasets/Air+Quality) as an example.
+"""
+
+import pandas as pd
+import tensorflow as tf
+dataset = tf.keras.utils.get_file(
+    fname='AirQualityUCI.csv',
+    origin="https://archive.ics.uci.edu/ml/machine-learning-databases/00360/"
+    "AirQualityUCI.zip",
+    extract=True
+)
+
+dataset = pd.read_csv(dataset, sep=';')
+dataset = dataset[dataset.columns[:-2]]
+dataset = dataset.dropna()
+dataset = dataset.replace(',', '.', regex=True)
+
+val_split = int(len(dataset) * 0.7)
+data_train = dataset[:val_split]
+validation_data = dataset[val_split:]
+
+data_x = data_train[['CO(GT)', 'PT08.S1(CO)', 'NMHC(GT)', 'C6H6(GT)',
+                     'PT08.S2(NMHC)', 'NOx(GT)', 'PT08.S3(NOx)', 'NO2(GT)',
+                     'PT08.S4(NO2)', 'PT08.S5(O3)', 'T', 'RH']].astype('float64')
+
+data_x_val = validation_data[['CO(GT)', 'PT08.S1(CO)', 'NMHC(GT)', 'C6H6(GT)',
+                              'PT08.S2(NMHC)', 'NOx(GT)', 'PT08.S3(NOx)', 'NO2(GT)',
+                              'PT08.S4(NO2)', 'PT08.S5(O3)', 'T', 'RH']].astype(
+                                  'float64')
+
+data_y = data_train['AH'].astype('float64')
+
+data_y_val = validation_data['AH'].astype('float64')
+
+"""
+The second step is to run the [TimeSeriesForecaster](/time_series_forecaster).
+As a quick demo, we set epochs to 10.
+You can also leave the epochs unspecified for an adaptive number of epochs.
+"""
+import autokeras as ak
+
+predict_from = 1
+predict_until = 10
+lookback = 3
+clf = ak.TimeseriesForecaster(
+    lookback=lookback,
+    predict_from=predict_from,
+    predict_until=predict_until,
+    max_trials=1,
+    objective='val_loss'
+)
+
+clf.fit(x=data_x, y=data_y, validation_data=(data_x_val, data_y_val), batch_size=30,
+        epochs=10)
+
+predictions = clf.predict(data_x_val)
```

Comment:
print the shape of the `predictions`.
Also call evaluation to see and print the result with a readable message `'...'.format()`

Reply:
my predictions array looks like this: `array([], shape=(0, 1), dtype=float32)`
Have you encountered this issue before?

Reply:
Figured out the issue here. In the predict function, the y_pred is sliced to only output the predictions from "train_len + predict_from" to "train_len + predict_until". Therefore, if we use data_x_val(which has length less than the train_len) the function will return an empty array. I'll try to figure out what makes the most sense here.

Note: Another thing that might cause some issues with the slicing is that the length of the y_pred before slicing is equal to "test_len - look_back + 1". Sp a very large lookback might result in an empty prediction array again.

Reply:
So based on the how the api is expected to be used we are currently going to enforce, that the predict data needs to contain the original train data time steps along with the time steps for which the prediction needs to be done.